src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -298,7 +297,7 @@ export class QuickBar extends LitElement {
   private _generateEntityItems(): QuickBarItem[] {
     return Object.keys(this.hass.states)
       .map((entityId) => ({
-        text: computeStateName(this.hass.states[entityId]),
+        text: computeStateName(this.hass.states[entityId]) || entityId,
```

Comment:
`computeStateName` should return the second part of the `entity_id` (after the dot) when there is no friendly name, so that this doesn't return anything is weird, does the state exist?

Reply:
The issue occurs when friendly name exists but is an empty string.

https://github.com/home-assistant/frontend/blob/89b07ea0ae79bf7199a31d70e2f757121c78a789/src/common/entity/compute_state_name.ts#L4-L8

I could update that method to be this?

```
export const computeStateName = (stateObj: HassEntity): string => {
  return stateObj.attributes.friendly_name
    ? computeObjectId(stateObj.entity_id).replace(/_/g, " ")
    : stateObj.attributes.friendly_name || "";
};
```

Reply:
Here's some examples from my production instance:

![image](https://user-images.githubusercontent.com/1480827/96902073-e5c49a80-1448-11eb-936b-20b1d7e530ce.png)

Reply:
Did you set those friendly names yourself? Otherwise, I think that is a bug with the unifi integration

Reply:
I agree it's a bug. But I still think the list items should gracefully handle that possibility?

I can also make it happen to any entity through customizations or the state editor.

Reply:
Yeah, you're are supposed to be able to make the name blank, but it shouldn't be the default I think. Anyway it is ok.